src/logger.ts

```diff
@@ -0,0 +1,26 @@
+import debug from 'debug';
+
+// Create debug instances for different parts of the application
+const cliDebug = debug('e18e:cli');
+const fileSystemDebug = debug('e18e:filesystem');
```

Comment:
Maybe this should be `e18e:cli:filesystem`? So the cli logs include it but you can narrow down to it still

Reply:
good call ✍🏻